src/openloco/windows/title_exit.cpp

```diff
@@ -0,0 +1,92 @@
+#include "../graphics/colours.h"
+#include "../graphics/gfx.h"
+#include "../interop/interop.hpp"
+#include "../intro.h"
+#include "../localisation/string_ids.h"
+#include "../openloco.h"
+#include "../ui.h"
+#include "../windowmgr.h"
+
+using namespace openloco::interop;
+
+namespace openloco::ui::windows
+{
+    namespace widx
+    {
+        enum
+        {
+            exit_button
+        };
+    }
+
+    static widget_t _widgets[] = {
+        make_widget( { 0, 0 }, { 40, 28 }, widget_type::wt_9, 1, string_ids::null, string_ids::title_menu_exit_from_game ),
```

Comment:
`wt_9` doesn't use strings. `-1` is a value that determines how it should be drawn. (Yes, it's messy)